tiers/revisions.go

```diff
@@ -0,0 +1,38 @@
+package tiers
+
+import (
+	"fmt"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-e2e/wait"
+	"github.com/stretchr/testify/require"
+)
+
+type Revisions map[string]string
+
+func GetRevisions(awaitility *wait.Awaitility, tier string, nsTypes ...string) (Revisions, error) {
+	templateTier, err := awaitility.Host().WaitForNSTemplateTier(tier, wait.UntilNSTemplateTierSpec(wait.Not(wait.HasNamespaceRevisions("000000a"))))
+	if err != nil {
```

Comment:
what about `required.NoError(awaitility.T, err)` instead?

Reply:
yeah, could be - it was copy-pasted from the previous location :-) 
changed in c7e22d6